.travis.yml

```diff
@@ -14,13 +14,21 @@ matrix:
       dist: trusty
       sudo: required
       env: SWIFT_SNAPSHOT=4.0.3
+    - os: linux
+      dist: trusty
+      sudo: required
+      env: SWIFT_SNAPSHOT=swift-4.2-DEVELOPMENT-SNAPSHOT-2018-07-01-a
     - os: osx
       osx_image: xcode9.3
       sudo: required
```

Comment:
Can you change this from Xcode9.3 to Xcode9.4, 9.4 comes with Swift 4.1.2 and means there wouldn't be a download for this build.

Reply:
I have added separate ``` os```  section for Xcode9.4 with the exsiting  versions .Do you want me to remove Xcode9.3 section completely ?